packages/quantic/cypress/integration/tab/tab-selectors.ts

```diff
@@ -4,13 +4,18 @@ export const tabComponent = 'c-quantic-tab';
 
 export interface TabSelector extends ComponentSelector {
   tab: () => CypressSelector;
+  button: () => CypressSelector;
   active: () => CypressSelector;
 }
 
 export const TabSelectors: TabSelector = {
   get: () => cy.get(tabComponent),
 
   tab: () => TabSelectors.get().find('li.slds-tabs_default__item'),
+  button: () =>
+    TabSelectors.get().find(
+      'button.pill__container.slds-grid_vertical-align-center.slds-p-horizontal_x-small.slds-p-vertical_xx-small'
```

Comment:
At first glance this selector seems overly complicated. I feel like `.pill__container` would be enough.
otherwise if we change anything about the classes on this component we'll break the tests.

Reply:
can we use selector based on the tab title or add another attribute for the "test" itself, I don't think it's a good practice for test to keep the selector this long.